pkg/apis/extensions/types.go

```diff
@@ -470,7 +511,7 @@ const (
 
 // JobCondition describes current state of a job.
 type JobCondition struct {
-	// Type of job condition, currently only Complete.
+	// Type of job condition.
```

Comment:
I prefer to keep anything unrelated to this feature in another PR, even pedantic stuff, but that's just IMO